proto/greptime/v1/region/server.proto

```diff
@@ -26,14 +26,24 @@ import "greptime/v1/ddl.proto";
 
 service Region { rpc Handle(RegionRequest) returns (RegionResponse); }
 
+message QueryContext {
```

Comment:
I think maybe we don't need that all fields in `QueryContext` to be in proto. I suggest:
```proto
message QueryContext {
  string current_catalog = 1;
  string current_schema = 2;
  // current_user is not needed
  timezone = 3;
  // sql_dialect is not needed
  map<string, string> extension = 4;
  // configuration_parameter is not needed
}

Reply:
Actually I thought the same, just wondering if we need to completely re-construct the context in the region server.

PR updated.